sample/app/index.tsx

```diff
@@ -11,13 +11,13 @@ export default function App() {
   return (
```

Comment:
The formatting in this file is a bit weird. Can we run prettier on it?

Reply:
I've been running `npm run format` so this is what it applied

Reply:
`npm run format` only formats the `src` directory, so it won't fix issues here.